guide/payments/payment-request-formats.md

```diff
@@ -0,0 +1,147 @@
+---
```

Comment:
I think this page would be better suited as being apart of the `Requesting bitcoin` page I am working on. We should re-order the pages also so that requesting is second after transactions. Users will likely request funds before making a payment so that order makes more sense to me.

Reply:
Let's take it out of the sub-sub-folder and make it a regular page in the Payments section.

Reply:
Yep I agree with both those suggestions.